packages/core/src/utilities/isViewportPreScaled.ts

```diff
@@ -0,0 +1,19 @@
+import { IViewport } from '../types';
+import { StackViewport, VolumeViewport } from '../RenderingEngine';
+import cache from '../cache';
+
+function isViewportPreScaled(viewport: IViewport, volumeId?: string) {
```

Comment:
we had this already no? packages/tools/src/utilities/viewport/isViewportPreScaled.ts